src/not_nintendogs/sprites.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
```

Comment:
https://github.com/letsbuilda/not-nintendogs/blob/d6e724fcf4f44990ca333130610ff5b1063f0ff7/pyproject.toml#L9
If we're already requiring a minimum of Python 3.10, what do we need the futures import for?